app/models/course.rb

```diff
@@ -304,4 +305,11 @@ def ensure_required_params
     self.timeline_start ||= start
     self.timeline_end ||= self.end
   end
+
+  def set_end_times
```

Comment:
Add documentation of the relationship between this the `use_start_and_end_times` method on course types, and how we interpret it to mean overriding the default of starting at 00:00 and ending at 23:59.